src/main/java/bloop/TaskList.java

```diff
@@ -0,0 +1,149 @@
+package bloop;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Operations related to the list of tasks.
+ */
+public class TaskList {
+
+    /** list of tasks */
+    private static ArrayList<Task> tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Constructor for TaskList object.
+     *
+     * @param ui Object of class Ui.
+     * @param storage Object of class Storage.
+     */
+    public TaskList(Ui ui, Storage storage) {
+        this.ui = ui;
+        tasks = new ArrayList<>();
```

Comment:
Did you intend to overwrite the static `tasks` field every time a new `TaskList` is created?